pyclibrary/c_parser.py

```diff
@@ -34,22 +34,29 @@
      WordEnd, Keyword, Regex, Literal, SkipTo, ZeroOrMore, OneOrMore,
      Group, LineEnd, stringStart, quotedString, oneOf, nestedExpr,
      delimitedList, restOfLine, cStyleComment, alphas, alphanums, hexnums,
-     lineno, Suppress)
+     lineno, Suppress, NoMatch, ParseException)
 ParserElement.enablePackrat()
 
 logger = logging.getLogger(__name__)
 
+__all__ = ['win_defs', 'CParser', 'MSVCParser', 'SYS_HEADER_DIRS']
```

Comment:
Oh. Missed this line, when changing the name to 'win_defs_parser'. Next commit will fix this.